sql/execution_pipeline.cc

```diff
@@ -378,12 +389,13 @@ from(std::shared_ptr<TableExpression> from,
      WhenExpression when,
      SelectExpression select,
      std::shared_ptr<SqlExpression> where,
-     OrderByExpression orderBy)
+     OrderByExpression orderBy,
+     GetParamInfo getParamInfo)
 {
-    return std::make_shared<FromElement>(shared_from_this(), from,
+    return std::make_shared<FromElement>(shared_from_this(), from, 
                                          BoundTableExpression(),
                                          when,
-                                         select, where, orderBy);
+                                         select, where, orderBy, getParamInfo);
```

Comment:
Would be worth a comment to describe why we need params in the table expression.  It's not obvious.